models/issue_reaction.go

```diff
@@ -69,6 +69,7 @@ func findReactions(e Engine, opts FindReactionsOptions) ([]*Reaction, error) {
 	reactions := make([]*Reaction, 0, 10)
 	sess := e.Where(opts.toConds())
 	return reactions, sess.
+		In("reaction.type", setting.UI.Reactions).
```

Comment:
```suggestion
		In("reaction.`type`", setting.UI.Reactions).
```

Reply:
this is pointles - if so i have to change the rest too - it works without :D just code format

Reply:
@axifive i can add if but otherwise i resolve

Reply:
@6543 I commented above

Reply:
@axifive i dont see `added value` but if you can explain why i should add "`" I'll do :D

Reply:
@axifive I wrote you a longer text on discord - to dont spam github

Reply:
sort i mean: [link](https://github.com/go-gitea/gitea/pull/9387/files#diff-61ce589b0f29ee34fc2448fcda3af467L72)

Reply:
`type` is reserved keyword so it is safer to escape it to be on safe side on all databases and versions :)

Reply:
Thx - missing part 👍